<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_SignUp</class>
 <widget class="QDialog" name="Window_SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QLabel{
font: 75 italic 9pt &quot;Comic Sans MS&quot;;
color:rgb(0, 170, 127)
}
QPushButton{
font: 75 italic 9pt &quot;Comic Sans MS&quot;;
color:rgb(0, 170, 127);

}
#Window_SignUp{
background-image: url(:/resource/back7.png);
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>191</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="sign_Layout">
    <item>
     <widget class="QPushButton" name="signIn">
      <property name="text">
       <string>sign in</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signUp">
      <property name="baseSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>sign up</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>301</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="username_Layout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="userName_Label">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>9</pointsize>
          <weight>9</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>UserName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="password_Label">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>9</pointsize>
          <weight>9</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="password_Again_Label">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>9</pointsize>
          <weight>9</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Password Again</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="userName_Text"/>
      </item>
      <item>
       <widget class="QLineEdit" name="password_Text"/>
      </item>
      <item>
       <widget class="QLineEdit" name="password_Again_Text"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_NotSame">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#password_NotSame{
color:rgb(255, 85, 127)}</string>
   </property>
   <property name="text">
    <string>not same</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorInfo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Username or password can't be empty.</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="showPassword">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#showPassword{
font: 75 italic 9pt &quot;Comic Sans MS&quot;;
color:rgb(60, 120, 100)
}</string>
   </property>
   <property name="text">
    <string>show</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>331</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="welcome_info1">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>16</pointsize>
        <weight>9</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#welcome_info1{
	
	font: 75 italic 16pt &quot;Comic Sans MS&quot;;
	color: rgb(69, 255, 81);
}</string>
      </property>
      <property name="text">
       <string>Welcome to</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="welcome_info2">
      <property name="font">
       <font>
        <family>Minion Pro</family>
        <pointsize>18</pointsize>
        <weight>9</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#welcome_info2{
color: rgb(170, 255, 0);
font: 75 italic 18pt &quot;Minion Pro&quot;;
}</string>
      </property>
      <property name="text">
       <string>Pokemon </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="welcome_info3">
      <property name="styleSheet">
       <string notr="true">#welcome_info3
{
	
	font: 75 italic 16pt &quot;Comic Sans MS&quot;;
	color: rgb(69, 255, 81);
}</string>
      </property>
      <property name="text">
       <string>of Thief !</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
